### PR TITLE
Sentinel data forwarder

### PR DESCRIPTION
**Pull request by sylviamclaughlin**
# Summary 

Github action that takes data from a previous github action passed in as input_data and send it to Sentinel. If  a file_name is provided as as input, then the Github action reads the contents of each file and sends it to Sentinel.

### CODE REVIEW

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/anaconda
+{
+	"name": "Anaconda (Python 3)",
+	"build": { 
+		"context": "..",
+		"dockerfile": "Dockerfile"
```
Comment (maxneuvians) [inline .devcontainer/devcontainer.json:7]: The only comment i have is that if you might nod need the `Dockerfile` file as long as you just define everything you need in the `features` property. There should be a python3 feature https://github.com/devcontainers/features/tree/main/src/python. This is really the tiniest optimisation and should not block this PR, maybe we can just open an issue.

File: src/Makefile [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+install:
+	pip3 install --user -r requirements.txt
+
+install-dev:
+	pip3 install --user -r requirements_dev.txt
+
+lint:
+	flake8 .
+
+test:
```
Comment (maxneuvians) [inline src/Makefile:15]: Could you add a CI file to run the tests on PR? Should be an example here: https://github.com/cds-snc/sre-bot/blob/main/.github/workflows/ci_code.yml

Comment (sylviamclaughlin) [reply src/Makefile:15]: Done. [CI file ](https://github.com/cds-snc/sentinel-forward-data-action/blob/sentinel_data_forwarder/.github/workflows/ci_code.yml) added and you can see the[ github action](https://github.com/cds-snc/sentinel-forward-data-action/actions/runs/3878260627) running and executing successfully.

File: .devcontainer/devcontainer.json (hunk 1 above)
Comment (sylviamclaughlin) [reply .devcontainer/devcontainer.json:7]: Thanks Max! I do have to say that VS Code devcontainers are a bit unfamiliar.  I have created an issue [here](https://github.com/cds-snc/sentinel-forward-data-action/issues/3).

**Review by maxneuvians — APPROVED** [review]
Nice!

**Pull request merged**